pykafka/simpleconsumer.py

```diff
@@ -777,12 +788,18 @@ def __init__(self, partition, handler=None, semaphore=None, compacted_topic=Fals
         :type compacted_topic: bool
         """
         self.partition = partition
+        self._consumer_id = consumer_id
         self._messages = Queue()
         self._messages_arrived = semaphore
         self._is_compacted_topic = compacted_topic
         self.last_offset_consumed = -1
         self.next_offset = 0
         self.fetch_lock = handler.RLock() if handler is not None else threading.RLock()
+        # include consumer id in offset metadata for debugging
+        self._offset_metadata = {
+            'consumer_id': self._consumer_id,
+            'hostname': socket.gethostname()
```

Comment:
Sure, I'll make that change. `self._consumer_id` does already contain `socket.gethostname()` under normal circumstances, so now we'll have both!